docs/feature-gates.md

```diff
@@ -100,8 +97,7 @@ for more information about TopologyAwareHints.
 
 #### Requirements for this Feature
 
-- `AntreaProxy` is enabled.
-- `EndpointSlice` is enabled.
+Feature `EndpointSlice` is enabled.
```

Comment:
ditto

Reply:
Any context for this ditto?

Reply:
I mean the removed line "- `AntreaProxy` is enabled" , if it still requires antreaProxy.enable to true, we may need to highlight this.

Reply:
For TopologyAwareHints, it still requires `requires antreaProxy.enable` to be true. I have added related text.